.surge.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
```

Comment:
Please add a comment stating what the purpose of this script is, and some link to help people not familiar with the service. My guess:
> Build the documentation and publish it on a public site
> It uses Mordor, https://mordor.mil/how_to_join